source/imaer-gml/src/main/java/nl/overheid/aerius/gml/base/characteristics/OPSSpreadUtil.java

```diff
@@ -42,27 +41,22 @@ public OPSSpreadUtil(final boolean legacySpread) {
   }
 
   public Double getSpread(final IsGmlBaseOPSSourceCharacteristics characteristics, final Double defaultSpread, final Geometry geometry) {
-    final Double spread = characteristics.getSpread();
-    final double emissionHeight = characteristics.getEmissionHeight();
+    final boolean pointLine = geometry instanceof Point || geometry instanceof LineString;
 
-    if (legacySpread) {
-      if (geometry instanceof Point || geometry instanceof LineString) {
-        return getSourceSpread(null, emissionHeight, defaultSpread);
+    if (pointLine) {
+      if (legacySpread) {
+        return 0.0;
       } else {
-        return getLegacySpread(characteristics, defaultSpread);
+        return getSpreadOrDefault(characteristics, 0.0);
```

Comment:
Shouldn't it be `defaultSpread` for the last argument? If it's a 6.0 version, but old enough AERIUS version, it'll be 0.0 anyway due to `legacySpread`, and if it's newer it makes more sense to use the supplied default as is done for polygons?

Reply:
Myeah, I guess that could be the case, or someone creating GMLs outside of our products and not adjusting the spread. And in that case, the default from the sector makes just as much sense as 0 I imagine (just a bit weird that it works different if the geometry is a polygon). Let's stick to 0.